store/modules/getters/user.js

```diff
@@ -34,7 +37,8 @@ export const getUserLikeCoinAmountIsZero = state => (
 );
 
 export const getUserHasERC20LikeCoin = state => !!(state.likecoinAmountObject
-  && state.likecoinAmountObject.walletLIKE && state.likecoinAmountObject.walletLIKE !== '0');
+  && state.likecoinAmountObject.walletLIKE
+  && new BigNumber(state.likecoinAmountObject.walletLIKE)).gt(ETH_MIGRATION_MIN_LIKECOIN_AMOUNT);
```

Comment:
The condition in migration page is `toBN(value).lt(toBN(ETH_MIN_LIKECOIN_AMOUNT))`, so here should use `gte`